config/checkstyle/checkstyle.xml

```diff
@@ -400,7 +400,7 @@
     <!-- Checks that each non-empty line in Javadoc has whitespace after leading asterisk. -->
     <module name="JavadocMissingWhitespaceAfterAsterisk"/>
 
-    <!-- Checks that for block tags, indentation of continuation lines is at least 4 spaces. -->
+    <!-- Checks that for block majors, indentation of continuation lines is at least 4 spaces. -->
```

Comment:
These should not be changed under checkstyle.xml

Reply:
Ok thank you it has been resolved.